tracer/test/Datadog.Trace.TestHelpers/TracerHelper.cs

```diff
@@ -39,8 +41,14 @@ public static ScopedTracer CreateWithFakeAgent(
 
         public class ScopedTracer : Tracer, IAsyncDisposable
         {
-            public ScopedTracer(TracerSettings settings = null, IAgentWriter agentWriter = null, ITraceSampler sampler = null, IScopeManager scopeManager = null, IDogStatsd statsd = null)
-                : base(settings, agentWriter, sampler, scopeManager, statsd)
+            public ScopedTracer(
+                TracerSettings settings = null,
+                IAgentWriter agentWriter = null,
+                ITraceSampler sampler = null,
+                IScopeManager scopeManager = null,
+                IDogStatsd statsd = null,
+                IDiscoveryService discoveryService = null)
+                : base(settings, agentWriter, sampler, scopeManager, statsd, discoveryService: discoveryService)
```

Comment:
Adding the parameter name to only argument just adds noise imo.
```suggestion
                : base(settings, agentWriter, sampler, scopeManager, statsd, discoveryService)
```

Reply:
Here named argument is used for define the position (positional argument)

the original definition is
https://github.com/DataDog/dd-trace-dotnet/blob/0e0269a25a3b53d606aafbbe0a7c514e93c3ec71/tracer/src/Datadog.Trace/Tracer.cs#L94-L97